File: elevatorsystem/src/main/java/com/elevator/elevatorsystem/elevator/controller/dto/LiftDto.java
package com.elevator.elevatorsystem.elevator.controller.dto;

import lombok.Data;

import java.util.List;

@Data
public class LiftDto {
    private String id;
    private String name;
    private String status;
    private Integer currentFloorNumber;
    private String currentFloorId;
    private List<Integer> floorStops;
    private String buildingId;

}

File: ElevatorSystemDotNet/Controllers/Dtos/LiftDto.cs
using System;

namespace ElevatorSystemDotNet.Controllers.Dtos;

public class LiftDto
{
    public string Id { get; set; }
    public string Name { get; set; }

    public string Status { get; set; }

    public int CurrentFloorNumber { get; set; }
    public string CurrentFloorId { get; set; }

    public List<int> FloorStops { get; set; } = [];

    public string BuildingId { get; set; }
}
